manifests/static/kruize-ui-nginx-setup.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-config
+  namespace: monitoring
+data:
+  nginx.conf: |
+    events {}
+    http {
+      upstream kruize-api {
+        server kruize:8080;
+      }
+
+      server {
+        listen 80;
+        server_name localhost;
+
+        location / {
+          root   /usr/share/nginx/html;
+          index  index.html index.htm;
+        }
+
+        location /api/ {
+          rewrite ^/api(.*)$ $1 break;
+          proxy_pass http://kruize-api;
+        }
+      }
+    }
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kruize-ui-nginx-service
+  namespace: monitoring
+spec:
+  type: NodePort
+  ports:
+    - name: http
+      port: 80
+      targetPort: 80
+      nodePort: 32323
+  selector:
+    app: kruize-ui-nginx
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kruize-ui-nginx-pod
+  namespace: monitoring
+  labels:
+    app: kruize-ui-nginx
+spec:
+  containers:
+    - name: kruize-ui-nginx-container
+      image: bharathappali/kruize-ui:test
```

Comment:
Image should be from the kruize repo

Reply:
Made the image as quay.io/kruize/kruize-ui:test